core/src/main/java/nl/nn/adapterframework/jdbc/JdbcIteratingPipeBase.java

```diff
@@ -47,7 +47,7 @@
 
 	protected MixedQuerySender querySender = new MixedQuerySender();
 
-	private final String FIXEDQUERYSENDER = "nl.nn.adapterframework.jdbc.FixedQuerySender";
+	private final String FIXEDQUERYSENDER = "nl.nn.adapterframework.jdbc.JdbcQuerySenderBase";
```

Comment:
Dit is niet de bedoeling, alleen maar een workaround.
De Ibisdoc generator moet de documentatie ook kunnen vinden als er naar FixedQuerySender verwezen wordt.